apps/tangle-dapp/components/ValidatorSelectionTable/ValidatorSelectionTable.tsx

```diff
@@ -278,7 +278,7 @@ const ValidatorSelectionTable: FC<ValidatorSelectionTableProps> = ({
           ) : (
             <Table
               tableClassName={cx('[&_tr]:[overflow-anchor:_none]')}
-              thClassName="border-t-0 py-3 sticky top-0"
+              thClassName="z-10 border-t-0 py-3 sticky top-0"
```

Comment:
Should the client need to set `z-index` with the `Checkbox` component, we should update the component to eliminate this requirement.